website/blog/2022-06-21-version-069.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Announcing React Native 0.69
+authors: [fortmarek, cortinico]
+tags: [announcement, release]
+---
+
+# Announcing 0.69
+
+Hey all, 👋
+
+We are excited to release a new version of React Native, 0.69.0. This version comes with several improvements for the New Architecture of React Native and new features: React 18 support & bundled Hermes. Read on to learn more!
+
+### Sections
+
+- [React 18](/blog/2022/06/21/version-069#react-18)
+- [Bundled Hermes](/blog/2022/06/21/version-069#bundled-hermes)
+- [Highlights of 0.69](/blog/2022/06/21/version-069#highlights-of-068)
+
+<!--truncate-->
+
+## React 18
+
+We are excited to share with you that React Native 0.69 is the first release to support React 18. React 18 has brought [lots of improvements](https://reactjs.org/blog/2022/03/29/react-v18.html), like new hooks such as `useId`. Additionally, React 18 includes new concurrency features such as `useTransition` or full Suspense support.
+
+On React Native 0.69, React 18 is enabled by default. However, if you have not migrated to the New Architecture, you will only be able to leverage the features that don't use concurrent rendering and concurrent features. New Architecture has been built with concurrent rendering in mind but we cannot add that support for the old architecture.
```

Comment:
"New Architecture has been built". Should this have a "The"? 

"The New Architecture has been built"

Reply:
Yeah, you are probably right, will change this 👍